File: test/Font/test04.py
#!/usr/bin/python

from ming import *
import sys

srcdir=sys.argv[1]

m =  SWFMovie();

font = SWFFont(srcdir + "/../Media/font01.fdb")

t = SWFText(1)
t.setFont(font)
t.setColor(0,0,255,255)
t.setHeight(20)
t.moveTo(100, 100)
t.addString("1234567890")

tf = SWFTextField()
tf.setFont(font)
tf.setColor(255,0,0,255)
tf.setHeight(20)
tf.addString("1234567890")

m.add(t);
i = m.add(tf);
i.moveTo(100, 120);

m.nextFrame(); 

m.save("test04.swf");

File: test/Font/test04.php
#!/usr/bin/php -c.

<?php

$srcdir = $argv[1];
$mediadir = $srcdir . "/../Media";

$m = new SWFMovie();

$f = new SWFFont($mediadir . "/font01.fdb");

$t = new SWFText(1);
$t->setFont($f);
$t->setHeight(20);
$t->setColor(0x00, 0x00, 0xff);
$t->moveTo(100, 100);
$t->addString("1234567890");

$tf = new SWFTextField();
$tf->setFont($f);
$tf->setHeight(20);
$tf->setColor(0xff, 0x00, 0x00);
$tf->addString("1234567890");

$m->add($t);
$it = $m->add($tf);
$it->moveTo(100, 120);

$m->nextFrame(); /* end of frame 1 */

$m->save("test04.swf");
?>
